ParcelGo/urls.py

```diff
@@ -15,8 +15,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("", include("user.urls")),
```

Comment:
user to raczej nie Twoja glowan czesc aplikacji wiec nie powinna byc pod ""
```suggestion
    path("autchentication/", include("user.urls")),
    albo tak
    path("users/", include("user.urls")),
```